src/index.ts

```diff
@@ -445,6 +444,7 @@ export class Bigtable {
       {},
       baseOptions,
       {
+        gaxServerStreamingRetries: true,
```

Comment:
This ensures that the data client will be initialized with options that have `gaxServerStreamingRetries` set to true and this will make `readRows`, `sampleRowKeys`, `mutateRows`, `generateInitialChangeStreamPartitions` and `readChangeStream` calls all do streaming calls with `gaxServerStreamingRetries` set to true.

Reply:
Maybe this would be useful as a comment in the code?